server/app/controllers/applicant/RedirectController.java

```diff
@@ -72,13 +70,6 @@ public RedirectController(
   public CompletionStage<Result> programBySlug(Http.Request request, String programSlug) {
     Optional<CiviFormProfile> profile = profileUtils.currentUserProfile(request);
 
-    if (profile.isEmpty()) {
```

Comment:
We currently ask users to log in whenever they go to a program slug. For example, open this in incognito:

https://civiform.seattle.gov/applicants/25936/programs/1384

Now, we just assume the user is a guest and take them to the program.

Reply:
question (blocking): Does the design not take into consideration users with accounts who are not logged in who land on a program slug?

Reply:
That sounds good to me but we should should confirm with @isabelcshaw and @sijiayam

Reply:
Resolved in an out-of-band discussion

Reply:
Follow up PR is tracked as part of https://github.com/civiform/civiform/issues/4705.